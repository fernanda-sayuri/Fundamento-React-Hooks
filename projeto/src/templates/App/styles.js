import styled, {css} from 'styled-components';

export const Wrapper = styled.div`
    //background: red;

    /* ${({ background }) => css`
        background = ${background};
    `} */
    
    ${({ theme }) => css`
        background = ${theme.colors.secondaryBg};
    `}
`;