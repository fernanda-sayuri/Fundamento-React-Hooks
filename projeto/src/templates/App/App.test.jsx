import { renderTheme } from '../../styles/render-theme'
import Home from '.';
import { theme } from '../../styles/theme';
import { screen } from '@testing-library/react'
// import 'jest-styled-components';


test('renders learn react link', () => {
  const { debug } = renderTheme(<Home />);
  debug();
  
  const headingContainer = screen.getByRole('heading', { name: 'Hello' })
  .parentElement;
  expect(headingContainer).toHaveStyle({
    background: theme.colors.mainBg,
  });
  expect(headingContainer).toMatchSnapshot();
  // expect(headingContainer).toHaveStyleRule('background', 'red');
});