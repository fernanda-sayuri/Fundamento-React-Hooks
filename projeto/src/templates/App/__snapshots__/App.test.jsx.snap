// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  class="sc-bczRLJ hFXWXT"
>
  <h1>
    Hello
  </h1>
</div>
`;
